src/libs/actions/IOU.ts

```diff
@@ -5062,7 +5062,7 @@ function updateMoneyRequestAmountAndCurrency({
     const transactionChanges = {
         amount,
         currency,
-        ...(taxAmount && {taxAmount}),
+        ...(taxAmount !== null && {taxAmount}),
```

Comment:
Same above

Reply:
Type is `taxAmount?: number` 🤔

Reply:
@MonilBhavsar In this case, I see `taxAmount: number` seems more accurate

Reply:
because we are using CurrencyUtils.convertToBackendAmount function to get taxAmount params and the returned type of this function is number

https://github.com/Expensify/App/blob/97097ff13b086c3d44d26d44335921323b211669/src/pages/iou/request/step/IOURequestStepAmount.tsx#L59-L67

Reply:
Ok cool, will update it

Reply:
@MonilBhavsar whoops, there still be a case `taxAmount` is **undefined**, It caused the TS error

Reply:
https://github.com/Expensify/App/blob/97097ff13b086c3d44d26d44335921323b211669/src/pages/iou/request/step/IOURequestStepAmount.tsx#L60-L62

we can `return 0` here or update the type taxAmount?: number to resolve this error

Reply:
done!